src/style.css

```diff
@@ -1 +1,87 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  max-width: 1200px;
```

Comment:
do not limit width of header - it should be always full-width
<img width="1440" alt="Screenshot 2022-08-23 at 23 03 17" src="https://user-images.githubusercontent.com/94289038/186255499-cb3e45be-46e1-4127-a35e-574378ad3191.png">